README.md

```diff
@@ -1 +1,69 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Leaderboard
+
+
+## Built With
+
+- Javascript
+- HTML and CSS
+
+
+[Live Demo Link](Not yet)
+
+
+## Getting Started
+
+**This project is part of a training series to highlight the need for proper documentation, best practices, and writing clean codes with html, css and Javascript.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a good working knowledge of Git and Github
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/wale-prog/To-do-list.git
+-To get the cuurent working tree, git fetch --all, then checkout to "todo-list-structure" branch.
+- Finally, display the content of the ./dist/index.html by starting the webpack dev server doing the following:
+navigate to the root directory with a command line terminal
+run 'npm start'
+
+### Install
+- IDE preferrably VScode
+- using command line terminal in the root directory install the following:
+     * node package manager with 'npm init -y'
+     * webpack with 'npm install webpack webpack-cli --save-dev'
+     *  HtmlWebpackPlugin with 'npm install --save-dev html-webpack-plugin'
+### Usage
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/yohannesdagnachew)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yohannes-dagnachew-5b163a236/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/yohannesdagnachew/To-Do-List/issues).
```

Comment:
- [ ] [OPTIONAL] Please, consider to fix the correct repo link - it seams this link is came from the old project "To do list"